content/introduction/eks/eks_high_level.md

```diff
@@ -0,0 +1,9 @@
+---
+title: "High Level"
+date: 2018-10-03T10:23:24-07:00
+draft: false
+weight: 150
+---
+
+
```

Comment:
Can we add few more details on this picture such as - now you have your cluster ready, how would you connect to your eks cluster, what is this experience etc?

Reply:
do you mean add more text? or adjust the picture?
What changes do you suggest?

Reply:
Text, start with what does this picture mean